apps/coordinator/ts/sessionKeys/__tests__/sessionKeys.service.test.ts

```diff
@@ -56,37 +82,30 @@ describe("SessionKeysService", () => {
     });
 
     test("should throw when given a non existent session key address", async () => {
+      const sessionKeysService = new SessionKeysService(fileService);
+
       const approval = await generateApproval(zeroAddress);
       await expect(
         sessionKeysService.generateClientFromSessionKey(zeroAddress, approval, ESupportedNetworks.OPTIMISM_SEPOLIA),
       ).rejects.toThrow(ErrorCodes.SESSION_KEY_NOT_FOUND.toString());
     });
 
     test("should generate a client from a session key", async () => {
-      jest.mock("@zerodev/sdk", (): unknown => ({
-        ...jest.requireActual("@zerodev/sdk"),
-        createKernelAccountClient: jest.fn().mockReturnValue({ mockedKernelClient: true }),
-      }));
+      const mockedAccount = { mockedKernelClient: true };
```

Comment:
Why do we need the `mockedKernelClient` attribute? 🤔

Reply:
I don't know, it was your code, I just updated mocks.

Reply:
These tests were in the maci-platform repo I did not write all the code. Let's leave it there then

Reply:
Sure, I'll check! I don't remember this one in maci-platform but probably can be fine to remove it.

Reply:
Ok, it works, I moved it to the mock initialization in `beforeEach` but keep the check of client to make sure and it returns the correct mocked one instead of real.